network/networkutils/networkutils_linux.go

```diff
@@ -176,6 +176,12 @@ func AllowIPAddresses(bridgeName string, skipAddresses []string, action string)
 	return nil
 }
 
+func BlockEgressTrafficFromContainer(ipAddress string, port int) error {
+	// iptables -t filter -I FORWARD -j DROP -d <ip>/32 -p tcp -m tcp --dport <port>
+	dropTraffic := fmt.Sprintf("-d %s/32 -p tcp -m tcp --dport %d", ipAddress, port)
+	return errors.Wrap(iptables.InsertIptableRule(iptables.V4, iptables.Filter, iptables.Forward, dropTraffic, iptables.Drop), "iptables block traffic failed")
```

Comment:
We are just blocking ipv4 ? Should we make the function intake the type of address as well ? (ipv4 or ipv6)

Reply:
there is no ipv6 wireserver endpoint. but i asked @QxBytes to create separate PR for blocking all ipv6 addrs from pod.